connectors/connectors.sh

```diff
@@ -7,6 +7,7 @@ set -euxo pipefail
 
```

Comment:
Should we deprecate this init action, instead of supporting it, as we natively support connectors upgrade in the DPGCE images now?

Reply:
The property names used here are still slightly different. E.g. Here we use spark-bigquery-connector-version, whereas in native support we use SPARK_BQ_CONNECTOR_VERSION. 
Sure, we can move these properties into native and deprecate this one.

It will still require us to make changes here so that it's at least no-op for 2.2 to avoid breakage (installation directory is different for 2.2).
I just felt this is easier.